libwebauthn/src/lib.rs

```diff
@@ -34,6 +36,20 @@ pub enum Transport {
     Ble,
 }
 
+#[derive(Debug)]
+pub enum StateUpdate {
+    UvRetry {
+        attempts_left: Option<u32>,
+    },
+    /// Oneshot channel
+    PinRequired {
```

Comment:
Could we add some simple docs to each of these variants? Here it would be helpful to explain the caller must use `reply_to`, and that a timeout may occur.